pallets/Polkadex_Ido/src/lib.rs

```diff
@@ -342,16 +342,14 @@ decl_module! {
             let max_amount = <WhiteListInvestors<T>>::get(round_id, investor_address.clone());
             ensure!(amount >= max_amount, Error::<T>::NotAValidAmount);
             ensure!(<InfoInvestor<T>>::contains_key(&investor_address), <Error<T>>::InvestorDoesNotExist);
-            <T as Config>::Currency::transfer(AssetId::POLKADEX, &investor_address, &Self::get_wallet_account(), amount)?;
-            let current_block_no = <frame_system::Pallet<T>>::block_number();
             let funding_round = <InfoFundingRound<T>>::get(round_id);
-            ensure!(current_block_no < funding_round.close_round_block && current_block_no > funding_round.start_block, <Error<T>>::NotAllowed);
             let total_raise = funding_round.amount.saturating_mul(funding_round.token_a_priceper_token_b);
             let investor_share = amount.checked_div(&total_raise).unwrap_or_else(Zero::zero);
+            let round_account_id = Self::round_account_id(round_id.clone());
+             <T as Config>::Currency::transfer(AssetId::POLKADEX, &investor_address, &round_account_id, amount)?;
             <InvestorShareInfo<T>>::insert(round_id, investor_address.clone(), investor_share);
             <InfoFundingRound<T>>::mutate(round_id, |round_details| {
-                let mut actual_raise = round_details.actual_raise;
-                actual_raise += amount;
+                round_details.actual_raise += amount;
```

Comment:
Just mentioning this since this PR is already merged, I recommend fixing this integer overflow in the next PRs which are planned to fix other IDO pallets issues. @mambisi

Reply:
The last version of the code was dead and did not even save the result I have added a ticket 

#282